cmd/wasm/assets/index.js

```diff
@@ -0,0 +1,87 @@
+const CEL_PROGRAM_DEFAULT_VALUE = `// Example CEL program
+bytes(get(state.url).Body).as(body, {
```

Comment:
This can be
```suggestion
get(state.url).Body.as(body, {
```
since we are using a modern runtime.

But the program could be even simpler
```
{
  "events": [get(state.url).Body.decode_json()]
}
```
if we wanted.

Reply:
The "less simpler" variant gives a little more "action" when you format it.

Reply:
Colour and movement. Gotit!